storage/blockchain/src/service/read.rs

```diff
@@ -601,3 +617,46 @@ fn alt_blocks_in_chain(env: &ConcreteEnv, chain_id: ChainId) -> ResponseResult {
 
     Ok(BlockchainResponse::AltBlocksInChain(blocks))
 }
+
+/// [`BlockchainReadRequest::Block`]
+fn block(env: &ConcreteEnv, block_height: BlockHeight) -> ResponseResult {
+    Ok(BlockchainResponse::Block(todo!()))
+}
```

Comment:
### TODO
All the new request/responses are fully typed but they will all panic with `todo!()`.

The exception is `AddressBookRequest::GetBan` which I added a new function for.

Handlers can be added in a different PR.